configs/configs.py

```diff
@@ -0,0 +1,11 @@
+
+# Discord Bot Settings
+DISCORD_TOKEN = ""
```

Comment:
Moving configs to this python file. This is to allow global setting variables across the project and makes it easier to retrieve instead of putting it in a json